src/Data/Lens/Lens/Unit.purs

```diff
@@ -5,5 +5,11 @@ import Prelude
 import Data.Lens.Lens (Lens', lens)
 
 -- | There is a `Unit` in everything.
+-- | ```purescript
+-- | > view united [1,2,3]
+-- | Unit
+-- | > over united (\a -> a) [1,2,3]
```

Comment:
Perhaps we show the type here to demonstrate that `\a -> a` here has to have the type `Unit -> Unit`?
```suggestion
-- | > over united (\a -> a :: Unit) [1,2,3]
```